portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,17 +32,28 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      
-    initializeComments();
     String json = convertToJsonUsingGson(commentList);
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
-  private void initializeComments() {
-    commentList.add("Awesome Website!");
-    commentList.add("Great job! Keep it up!");
-    commentList.add("Very interesting!!");
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String comment = getParameter(request, "comment-input", "");
+      commentList.add(comment);
+      response.sendRedirect("/index.html");
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
+   */
```

Comment:
_minor_ - it seems a bit odd that we have a return block in the comment section with no description. How about just having a description and do away with the `@return`. What do you think?

Reply:
Makes sense!